staging/kos/etcd-tls/cnf/etcd-client.cnf

```diff
@@ -0,0 +1,11 @@
+[req]
+req_extensions = v3_req
+distinguished_name = req_distinguished_name
+[req_distinguished_name]
+[ v3_req ]
+basicConstraints = CA:FALSE
+keyUsage = nonRepudiation, digitalSignature, keyEncipherment
+extendedKeyUsage = clientAuth, serverAuth
+subjectAltName = @alt_names
+[alt_names]
+DNS.1 = *
```

Comment:
This SAN makes this cert worthless for verifying the bearer's identity.

Reply:
See comment above.